### PR TITLE
Download Header preference.

### PR DESCRIPTION
**Pull request by Vjdevz**
Updated download header preference for AM.

### CODE REVIEW

**Review by Vjdevz — COMMENTED** [review]
@lindalu-MSFT

File: docs/actionable-messages/index.md [hunk 1]
```diff
@@ -112,6 +112,15 @@ Actionable messages are available to all customer mailboxes on Exchange Online i
 | Office Professional Plus 2016 (Click-to-Run only)    | Actionable Messages are available for Office 365 only                   | No |
 | Exchange 2016 On-Premises Outlook on the web         | Actionable Messages are available for Office 365 only                   | No |
 
+###Note:
```
Comment (lindalu-MSFT) [inline docs/actionable-messages/index.md:115]:
```suggestion
### Notes
```

File: docs/actionable-messages/index.md [hunk 2]
```diff
@@ -112,6 +112,15 @@ Actionable messages are available to all customer mailboxes on Exchange Online i
 | Office Professional Plus 2016 (Click-to-Run only)    | Actionable Messages are available for Office 365 only                   | No |
 | Exchange 2016 On-Premises Outlook on the web         | Actionable Messages are available for Office 365 only                   | No |
 
+### Notes
+Actionable messages may not render correctly in Outlook for windows if you have any of the following options enabled.
```
Comment (lindalu-MSFT) [inline docs/actionable-messages/index.md:116]:
```suggestion
Actionable messages may not render correctly in Outlook for Windows if you have any of the following options enabled.
```

File: docs/actionable-messages/index.md [hunk 3]
```diff
@@ -112,6 +112,15 @@ Actionable messages are available to all customer mailboxes on Exchange Online i
 | Office Professional Plus 2016 (Click-to-Run only)    | Actionable Messages are available for Office 365 only                   | No |
 | Exchange 2016 On-Premises Outlook on the web         | Actionable Messages are available for Office 365 only                   | No |
 
+### Notes
+Actionable messages may not render correctly in Outlook for Windows if you have any of the following options enabled.
+-Download Headers and then Full Items
```
Comment (lindalu-MSFT) [inline docs/actionable-messages/index.md:117]:
```suggestion
- Download Headers and then Full Items
```

File: docs/actionable-messages/index.md [hunk 4]
```diff
@@ -112,6 +112,15 @@ Actionable messages are available to all customer mailboxes on Exchange Online i
 | Office Professional Plus 2016 (Click-to-Run only)    | Actionable Messages are available for Office 365 only                   | No |
 | Exchange 2016 On-Premises Outlook on the web         | Actionable Messages are available for Office 365 only                   | No |
 
+### Notes
```
Comment (lindalu-MSFT) [inline docs/actionable-messages/index.md:115]:
```suggestion
> [!NOTE]
> Actionable messages may not render correctly in Outlook for Windows if you have any of the following options enabled in **Download Preferences** on the **Send / Receive** tab.
>
> - Download Headers and then Full Items
> - Download Headers
> - On Slow Connections Download Only Headers
>
> To resolve the problem, select **Download Full Items**. 
> Depending on your connection type, you may want to uncheck **On Slow Connections Download Only Headers** .
```

File: docs/actionable-messages/index.md [hunk 5]
```diff
@@ -112,6 +112,15 @@ Actionable messages are available to all customer mailboxes on Exchange Online i
...
+Actionable messages may not render correctly in Outlook for Windows if you have any of the following options enabled.
```
Comment (lindalu-MSFT) [inline docs/actionable-messages/index.md:116]:
```suggestion

```

File: docs/actionable-messages/index.md (hunk 3 above)
Comment (lindalu-MSFT) [inline docs/actionable-messages/index.md:117]:
```suggestion
```

File: docs/actionable-messages/index.md [hunk 6]
```diff
@@ -112,6 +112,15 @@ Actionable messages are available to all customer mailboxes on Exchange Online i
...
+-Download Headers and then Full Items
+-Download Headers or
```
Comment (lindalu-MSFT) [inline docs/actionable-messages/index.md:118]:
```suggestion
```

File: docs/actionable-messages/index.md [hunk 7]
```diff
@@ -112,6 +112,15 @@ Actionable messages are available to all customer mailboxes on Exchange Online i
...
+-On Slow Connections Download Only Headers, under Download Preferences in the Ribbon.
```
Comment (lindalu-MSFT) [inline docs/actionable-messages/index.md:119]:
```suggestion
```

File: docs/actionable-messages/index.md [hunk 8]
```diff
@@ -112,6 +112,15 @@ Actionable messages are available to all customer mailboxes on Exchange Online i
...
+
+To resolve the problem, navigate to Send /Receive tab in Outlook ribbon > click on Download Preferences and select **Download Full Items**.
```
Comment (lindalu-MSFT) [inline docs/actionable-messages/index.md:121]:
```suggestion
```

File: docs/actionable-messages/index.md [hunk 9]
```diff
@@ -112,6 +112,15 @@ Actionable messages are available to all customer mailboxes on Exchange Online i
...
 | Exchange 2016 On-Premises Outlook on the web         | Actionable Messages are available for Office 365 only                   | No |
 
+### Notes
+Actionable messages may not render correctly in Outlook for Windows if you have any of the following options enabled.
+-Download Headers and then Full Items
+-Download Headers or
+-On Slow Connections Download Only Headers, under Download Preferences in the Ribbon.
+
+To resolve the problem, navigate to Send /Receive tab in Outlook ribbon > click on Download Preferences and select **Download Full Items**. 
+You may want to uncheck **On Slow Connections Download Only Headers** depending on your connection type.
```
Comment (lindalu-MSFT) [inline docs/actionable-messages/index.md:122]:
```suggestion
```

**Review by lindalu-MSFT — CHANGES_REQUESTED** [review]

**Pull request merged**